plugins/inputs/system/linux_sysctl_fs.go

```diff
@@ -80,9 +89,10 @@ func (sfs *SysctlFS) Gather(acc telegraf.Accumulator) error {
 }
 
 func init() {
+
 	inputs.Add("linux_sysctl_fs", func() telegraf.Input {
 		return &SysctlFS{
-			path: "/proc/sys/fs",
+			path: path.Join(GetHostSys(), "/fs"),
```

Comment:
Yes, looks like we are not actually using HOST_SYS here.  I think the cgroups plugin is an example where it is needed.